docs/src/main/sphinx/admin/fault-tolerant-execution.rst

```diff
@@ -91,11 +92,31 @@ recommended when executing large batch queries, as the cluster can more
 efficiently retry smaller tasks within the query rather than retry the whole
 query.
 
-``TASK`` retry policy requires a configured :ref:`exchange manager
-<fte-exchange-manager>` to store spooled exchange data used for each task. It is
-also strongly recommended to set the ``query.low-memory-killer.policy``
-configuration property to ``total-reservation-on-blocked-nodes``, or queries may
-need to be manually killed if the cluster runs out of memory.
+The following cluster configuration changes are recommended to improve the
+efficacy of fault-tolerant execution with a ``TASK`` retry policy:
+
+* Set the ``query.low-memory-killer.policy``
+  :doc:`query management property </admin/properties-query-management>` to
+  ``total-reservation-on-blocked-nodes``, or queries may
+  need to be manually killed if the cluster runs out of memory.
+* Reduce the ``query.low-memory-killer.delay``
+  :doc:`query management property </admin/properties-query-management>`
+  so that the cluster can more quickly unblock nodes that run out of memory.
+* Modify the ``query.remote-task.max-error-duration``
+  :doc:`query management property </admin/properties-query-management>`
+  to adjust how long Trino allows a remote task to try reconnecting before
+  considering it lost and rescheduling.
+* Enable write scaling with the the ``scale-writers`` :doc:`configuration
```

Comment:
I think this is no longer true .. @arhimondr ?

Reply:
It's in the blog post PR, line 143: https://github.com/trinodb/trino.io/pull/287/files#diff-12a93c0c14ddbe839c036a547c49af77b3d2fc0fc6da6e369d1d551ca173e573R143

I myself am not too clear why write scaling is related to FTE, I added it because it was recommended in the post.

Reply:
We are about to land https://github.com/trinodb/trino/pull/10614 that enables scaled writers by default. I think we can drop this.

Reply:
Removed